src/test/java/gov/loc/repository/bagit/creator/BagCreatorTest.java

```diff
@@ -56,14 +56,25 @@ public void testBagInPlace() throws IOException, NoSuchAlgorithmException{
     Path file2Path = Paths.get(file2.toURI());
     file2.createNewFile();
     
+    
+    boolean isWindows = System.getProperty("os.name").contains("Windows");
```

Comment:
I like this & was thinking it would be good to hoist into a util file so it could be used everywhere else